docs/usage/help.rst.inc

```diff
@@ -109,7 +109,7 @@ Examples::
     $ borg create -e /home/user/cache/ backup /
 
     # The file '/home/user/cache/important' is *not* backed up:
-    $ borg create -e /home/user/cache/ backup / /home/user/cache/important
```

Comment:
it's maybe a bit weird example, but i guess it really was meant like that:
- the exclude is on `/home/user/cache/`
- the borg commandline args give 2 backup "roots":
  - `/` (meaning: everything below the root directory, except otherwise excluded)
  - `/home/user/cache/important` (meaning the file of this name)
- but the second one is not backed up, because it is in the excluded dir

Reply:
You're right